package.json

```diff
@@ -59,6 +58,22 @@
     "plugin:build:commit": "lerna run build:commit --ignore=\"@grafana/*\" --ignore=\"@grafana-plugins/input-datasource\"",
     "plugin:build:dev": "lerna run dev --ignore=\"@grafana/*\" --ignore=\"@grafana-plugins/input-datasource\""
   },
+  "nx": {
+    "targets": {
+      "start": {
+        "dependsOn": [
```

Comment:
We should add the Prometheus package here when it has a build step, shouldn't we?

Reply:
Exactly! I'd like us to reach consensus first on this approach before updating our google doc to highlight what needs to be done where to make sure plugins are decoupled without issue. 👍

Reply:
Reading this again, I thought you were refering to the extracted prometheus plugin not the package. The package won't need to be added here as this repository consumes its src files. Build is only required for publishing a package to the npm registry.